src/main/java/com/vendor/caterer/dto/MenuItemUpdateRequest.java

```diff
@@ -0,0 +1,24 @@
+package com.vendor.caterer.dto;
+
+import lombok.Data;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+@Data
+public class MenuItemUpdateRequest {
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private boolean isVeg;
+    private boolean isDrink;
+    private String imageUrl;
+    private List<String> tagsToAdd;
+    private List<String> tagsToRemove;
+    private Set<UUID> categoryIdsToAdd;
```

Comment:
Can it be updated to a model for  categoryIdsToAdd  like 
CateroryUpdate{
Operation type; // Add/Delete;
Set<UUID> ids;
}

Reply:
done